concent_api/api_testing_common.py

```diff
@@ -165,6 +165,8 @@ def _print_data(data, url):
     print()
     if response.status_code in [202, 204]:
         return None
+    elif response.status_code == 400:
+        return json.loads(response._content)
     else:
         return try_to_decode_golem_message(private_key, public_key, response.content)
```

Comment:
This is wrong. We shouldn't even try to decode the response as a Golem message if `Content-Type` is not `application/octet-stream`.

If you see something like this in other places, please fix it too.

Reply:
resolved in https://github.com/golemfactory/concent/pull/773